### ISSUE TITLE
Segmentation fault (core dumped) during SURVIVOR merge

### ISSUE THREAD

**Issue by sachingadakh** [fritzsedlazeck/SURVIVOR#176]
Hello, I am trying to merge 6 vcf files from different SV callers, called from long-read data. Each VCF file has SVs of around 40-75k.
However, I am getting a Segmentation fault (core dumped)  error as I am trying to merge them. I tried on sever as well to process it, but the error is persistent. Any help would be appreciated. Thank you
I used the following command :

survivor/SURVIVOR/Debug/SURVIVOR merge path/files 1000 3 1 1 0 50 output.vcf

**Comment by B0r1sD** [comment]
Are you perhaps making use of compressed VCF files (.vcf.gz)? Then you first have to unzip them first, as this is not a feature of SURVIVOR as mentioned by the author commented at the end of #72.

**Comment by leonvanmierlo** [comment]
Have you fixed this issue yet? I run into the same problem and my VCFs are not zipped.

**Comment by fritzsedlazeck** [comment]
Can you post the command and whats written in the terminal ? 
Would need to see at what stage this happens. It can for example also write permissions...

**Comment by leonvanmierlo** [comment]
@fritzsedlazeck Thank you for your reply. 
The command I used is: SURVIVOR merge sample_VCFlist 1000 1 0 0 0 0 sample_mergedWithSURVIVOR.vcf

The error it returns is:
merging entries: 12
SURVIVOR.sample.sh: line 22: 3970193 Segmentation fault      (core dumped) 

I try to merge a VCF from Manta and a VCF from Dysgu. Manta's VCF contains 12 SVs, Dysgu's VCF contains 342 SVs.

I hope this helps.

**Comment by fritzsedlazeck** [comment]
yeah seems like its failing on the Dysgu VCF file.. strange. I never saw that VCF file before.. 

honestly we also just pushed out Truvari that might be able to handle it. For that you do a bcftools merge on the two VCF files and then a truvari collapse ..

**Comment by leonvanmierlo** [comment]
Thank you for the suggestion. I will look into that as well.
What is also weird to me, if I run the same command multiple times, 1 in x times it will not get the segmentation fault (see attached photo).


![7569615F-4881-4917-877B-0A7D6E272157](https://github.com/fritzsedlazeck/SURVIVOR/assets/113985936/39712eb8-856e-4cc0-a2e1-db9574befa5e)

**Comment by nuttynutmore** [comment]
I also came across this error (because files were in .gz zipped format).  I am running SURVIVOR in a pipeline with hundreds of samples.

I have been running the command: 
`SURVIVOR merge sample1_input.survivor_sample_sheet.txt 1000 2 1 1 0 5 output.vcf` 

One of the inputs in the sample sheet is in .gz format.  The annoying thing is, the "Segmentation fault" error message only came up for one out of nearly 300 samples.

An error would be very helpful to identify this issue quickly.  Also, perhaps making this clearer in the documentation would help.
